src/services/functionAppService.ts

```diff
@@ -124,6 +117,7 @@ export class FunctionAppService extends BaseService {
     this.log("Deploying serverless functions...");    
 
     await this.zipDeploy(functionApp);
+    await this.uploadFunctionAppToBlobStorage();
```

Comment:
commented suggestion below but this name is kinda misleading, both are zip deploy, the only difference is zip storage location

Reply:
also why are we uploading to both place? what is the value of `RUN_FROM_PACKAGE`?